ext/clamrb/clamrb.c

```diff
@@ -33,7 +33,7 @@ VALUE scan(VALUE self, VALUE filename)
   Data_Get_Struct(self, struct cl_engine, engine);
   const char *name = NULL;
   long unsigned int scanned = 0;
-  int ret = cl_scanfile(StringValueCStr(filename), &name, &scanned, engine, CL_SCAN_STDOPT);
+  int ret = cl_scanfile(StringValueCStr(filename), &name, &scanned, engine, CL_DB_STDOPT);
```

Comment:
I checked the updated clamav docs and this doesn't solve the issue. While this is what the compiler says, the actual interface requires a scan options struct. If this is something you need I can make the updates.